routes/v1/profile.js

```diff
@@ -7,16 +7,15 @@ module.exports = wrap(async function (req, res) {
     const profileid = req.params.profileid;
     let uuid = req.params.uuid;
     if (!isUuid(uuid)) {
-        const mojang_response = await makeRequest(res, `https://api.ashcon.app/mojang/v2/user/${uuid}`);
-        if (mojang_response?.data?.uuid) {
-            uuid = mojang_response.data.uuid.replace(/-/g, '');
+        const mojang_response = await makeRequest(res, `https://playerdb.co/api/player/minecraft/${uuid}`);
+        if (mojang_response?.data?.data?.player?.raw_id) {
+            uuid = mojang_response?.data?.data?.player?.raw_id;
```

Comment:
Why was this changed?

Reply:
Oops, forgot to revert that. Was testing if playerdb API response was faster, doesn't really matter. Revert or keep, it's same thing

Reply:
I'm not really sure, I would revert probably.